src/types.d.ts

```diff
@@ -59,6 +59,13 @@ interface RCOResource {
 interface ActiveReferenceItem extends RCOResource {
   // when false, the item should not be included in drop-downs
   // for `create` forms, though it should for `edit` forms
+  id: number
+  name: string
+  active: boolean
+}
+
+interface ReferenceItem extends RCOResource {
+  id: string
   name: string
   active: boolean
 }
```

Comment:
Could we refactor this a little please?

I guess we'll have `ActiveItem` which as `name` and `active` fields.  Then we'll extend that with `ActiveItemInt` (with number id) and `ActiveItemString (with string id).